tests/const.py

```diff
@@ -303,6 +306,9 @@
         "walmart_exception",
         "zpackages_delivered",
         "zpackages_transit",
+        "ebay_delivering",
```

Comment:
```suggestion
```
Since eBay doesn't actually deliver packages we wouldn't add a "delivering" sensor.